.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: "[Setup] - Install GitVersion"
         uses: gittools/actions/gitversion/setup@v0.9.9
         with:
-          versionSpec: '5.6.6'
+          versionSpec: '5.7.0'
```

Comment:
This is not the reason. If I move it back to 5.6.6 still fails. I am not sure what changed.

Reply:
We figured out it was because the CLI application has the GitVersion NuGet package.  We don't need it at the GitHub Action will correctly set the project version.